automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSpec.scala

```diff
@@ -460,5 +468,36 @@ class LeonardoSpec extends FreeSpec with Matchers with Eventually with ParallelT
         }
       }
     }
+
+    "should allow BigQuerying in a new billing project" in withWebDriver { implicit driver =>
+      val ownerToken = hermioneAuthToken
+
+      withNewBillingProject { projectName =>
+
+        // project owners have the bigquery role automatically, so this also tests granting it to users
+
+        val ronEmail = LeonardoConfig.Users.ron.email
+        Orchestration.billing.addUserToBillingProject(projectName.value, ronEmail, Orchestration.billing.BillingProjectRole.User)(ownerToken)
+        Orchestration.billing.addGoogleRoleToBillingProjectUser(projectName.value, ronEmail, "bigquery.jobUser")(ownerToken)
+
+        implicit val leoToken: AuthToken = ronAuthToken
+        withNewCluster(projectName) { cluster =>
+          withNewNotebook(cluster) { notebookPage =>
+            val query = """! bq query "SELECT COUNT(*) AS scullion_count FROM publicdata.samples.shakespeare WHERE word='scullion'" """
```

Comment:
https://www.opensourceshakespeare.org/views/plays/play_view.php?WorkID=henry4p2&Act=2&Scene=1&Scope=scene

Reply:
I also considered rampallian and fustilarian.